tests/CMakeLists.txt

```
@@ -60,6 +60,9 @@ set(PYBIND11_TEST_OVERRIDE
 set(PYBIND11_TEST_FILTER
     ""
     CACHE STRING "Tests from ;-separated list of *.cpp files will be removed from all tests")
+set(PYBIND11_PYTEST_FLAGS
+    ""
+    CACHE STRING "Flags from ;-separatedlist to pass to pytest")
```

```suggestion
    CACHE STRING "Flags from ;-separated list to pass to pytest")
```

Check with space separated list, I believe that works too.